src/media/media.service.ts

```diff
@@ -157,6 +159,23 @@ export class MediaService {
     return mediaUpload;
   }
 
+  /**
+   * @async
+   * List all uploads by a specific user
+   * @param {User} user - the specific user
+   * @return {MediaUpload[]} arary of media uploads owned by the user
+   */
+  async listUploadsByUser(user: User): Promise<MediaUpload[]> {
+    const mediaUploads = await this.mediaUploadRepository.find({
+      where: { user: user },
+      relations: ['user', 'note'],
+    });
+    if (mediaUploads === undefined) {
+      return [];
```

Comment:
Codecov want's that this line is also covered by either an e2e or unit test.

Reply:
Added another test for that to `media.service.spec.ts`.